examples/fingerprint.py

```diff
@@ -0,0 +1,168 @@
+import deepchem as dc
+import json
+import numpy as np
+
+from copy import deepcopy
+from functools import partial
+from hyperopt import hp, fmin, tpe
+from shutil import copyfile
+from sklearn.ensemble import RandomForestClassifier
+
+from utils import mkdir_p, init_trial_path
+
+def decide_metric(dataset):
```

Comment:
Can we swap this to use the same `yapf` settings as DeepChem? Not a big deal, but it would be nice to keep all the DeepChem codebases stylistically similar

Reply:
Done.

Reply:
This is still 4 space while DeepChem is 2 space. Not a big deal but good to fix in a future PR